webproject/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -73,7 +74,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
             .getReference("messages")
```

Comment:
create function in firebaesUtil to getMessagesReference

Reply:
Updated